### PR TITLE
Remove MatomoService from the codebase

### PR DESCRIPTION
**Pull request by RandomAshes**
## Issue
Resolves N/A

## What was done
- [x] Remove MatomoService from the codebase

## How to test
1. Ensure that there is nothing related to matomo in the codebase anymore
2. Ensure the app's functionality has not changed at all

### CODE REVIEW

File: ui/src/Services/Api/SpaceClient.ts [hunk 1]
```diff
@@ -67,24 +66,12 @@ async function createSpaceForUser(spaceName: string): Promise<AxiosResponse<Spac
...
 
-async function editSpaceName(uuid: string, editedSpace: Space, oldSpaceName: string): Promise<AxiosResponse> {
-    return editSpace(uuid, editedSpace).then(result => {
-        MatomoService.pushEvent(oldSpaceName, 'editSpaceName', editedSpace.name);
-        return result;
-    }).catch(err => {
-        MatomoService.pushEvent(oldSpaceName, 'editSpaceNameError', editedSpace.name, err.code);
-        return Promise.reject(err);
-    });
+async function editSpaceName(uuid: string, editedSpace: Space): Promise<AxiosResponse> {
```
Comment (sapphon) [inline ui/src/Services/Api/SpaceClient.ts:69]: I especially like this vs. the previously-explicit `return Promise.reject(...)`

**Review by sapphon — COMMENTED** [review]
All files eyeball-reviewed; approval is currently gated by my inability to run the app locally, which should be resolved 8/1

File: ui/src/Services/Api/ProductClient.ts [hunk 2]
```diff
@@ -29,52 +27,23 @@ function getBaseProductsUrl(spaceUuid: string): string {
 
 async function createProduct(space: Space, product: Product): Promise<AxiosResponse> {
     const url = getBaseProductsUrl(space.uuid || '');
-    return Axios.post(url, product, getAxiosConfig()).then(result => {
```
[the remark this answers is not in the document]
Comment (RandomAshes) [reply ui/src/Services/Api/ProductClient.ts:32]: I agree. These files were way too busy.

**Review by hirava271 — APPROVED** [review]

**Review by sapphon — APPROVED** [review]
No calls to Matomo are made, and the app behaves normally!

**Pull request merged**